### ISSUE TITLE
Craft doesnt recognize plugin

### ISSUE THREAD

**Issue by xamo1998** [Logitech/logi_craft_sdk#4]
I made all the steps you said but I have a few doubts:

1. When we create the project with Pycharms what do we do with it?
2. I copy the manifest folder (The one with numbers) to the LogiOptionsPlugins but nothing happens.
3. When i try to select an app doesnt appear in the Add aplication option so i dont really know how to make it appear.
4.Where do I write what to do when anything happens?

Thanks a lot!

**Comment by naokihada** [comment]
Before official follow up, here's my knowledge. And which OS are you working on?
* It needs to be executable by itself. Python code need to be packaged to executable.
  Win: .exe
  Mac: .app Objc sample code document might be more easy to follow.
* Wrong format on manifest may not report anything. I use https://jsonlint.com/ for format verification.
* Check if it developer mode.
* Some case I needed to restart LogiOption service. (after copied manifest file, it is required)
   * Win: Kill Logi* services from Task Manager and run LogiOption UI to restart services
   * Mac: Force process to quite Logi Options Deamon from Activity Monitor. It will restart automatically. PID should be different.
* This issue page is monitor by official support.
Thank you.

**Comment by xamo1998** [comment]
Im working on windows.
I tried restarting Logitech options but nothing happens.
I will try to explain better:

1. I downloaded and installed Pycharms, and the webclient, and all the things I need.
    1.1. Create a project with Pycharms and copy the craft.py inside the root folder but I didnt use the 
          projects for nothing because it says nothing about what to do with the project.
2. I got my craft.exe with pyinstaller.
3. I created the folder LogiOptions Plugins and copy inside the folder with the manifest, gallery and language.
4. I opened the logitech Options (With developer mode) and try to go to all applications and then Add application but nothing appear. I'll let an image.

Sorry if my questions are so nobby...
Im trying to make a plugin for android studio.
![untitled](https://user-images.githubusercontent.com/28656786/38106543-97929034-338f-11e8-8806-c488f2193592.png)
